### DISCUSSION TITLE
Livewire actions get 302 response

### DISCUSSION THREAD

**Discussion by zakyd** [livewire/livewire#2818] [1. Help]
I have a laravel project deployed on a domain subfolder
- https://domain.com => another project
- https://domain.com/dashboard => my laravel project

I have set my livewire config `asset_url` to `config('app.url')`, which my `.env`'s `APP_URL` is `https://domain.com/dashboard`
It is a simple project that has an auth middleware which protect `https://domain.com/dashboard` and redirect it to `https://domain.com/dashboard/login`. And when I try to fill the input (it has wire:model attribute), I get an error at my JS console

`Uncaught (in promise) SyntaxError: Unexpected token < in JSON at position 0 at JSON.parse (<anonymous>) at index.js:58`

When I check the Network tab, there were some process:
- https://domain.com/dashboard/livewire/message/login (post) => 302 (i think it is not expected)
- https://domain.com/dashboard/login (get) => 200 (i think it makes the error above)

Could anybody help me?

==================

Edit:
It works fine if I deploy it to https://domain.com, but at https://domain.com/dashboard it keeps getting this issue

**Comment by cg0012** [comment]
Is "another project" also Laravel or with similar routing, whereas it is redirecting Route::get('/dashboard', function with a hard redirect)...(would that even work or just an endless circle??). I'm interested in how the web server knows how to serve this. 

I've got something similar but within a subdomain. "dashboard.domain.com". The web server file tree has a dashboard folder with the Laravel app to serve out of for the Laravel app. The regular domain has its own folder and is independent. Highly recommend this structure, if possible.

I don't know if this would be help at all - but you might add a tag in the head of your app layout blade view:

`<base href="https://domain.com/dashboard" />`

a .htaccess file pointing the requests properly between the "public" folders is most likely what's needed.

Good luck!

**Comment by jdbravo** [comment]
@zakyd did you find a solution for this problem? I upgraded filament from 2.13.2 to the version 2.16.52 without changing any file and I started having this problem.

When I try to login, it sends a request to admin/livewire/message/login but it receives a 302 code, redirecting it to admin/login (which is not JSON) and for that reason we are getting the JSON parse error.

For some reason filament is not accepting the request to admin/livewire/message/login but redirecting it to admin/login.

I was able to reproduce the problem with a fresh filament v2.16.52 project install and setting the assets_url to '/admin' in config/livewire.php and changing the path in config/filament.php to ''.

I'm using nginx to access filament with this config:
location /admin {
                alias /Users/jbravo/Documents/test-filament/example-app/public;
                try_files $uri $uri/ @admin;

                location ~ \.php$ {
                        fastcgi_split_path_info ^(.+\.php)(/.+)$;
                        fastcgi_index index.php;
                        include fastcgi_params;
                        fastcgi_param SCRIPT_FILENAME $request_filename;
                        fastcgi_pass   127.0.0.1:9000;
                }
}
location @admin {
                rewrite /admin/(.*)$ /admin/index.php last;
}

It's important to say that it was working without any issue on filament v2.13.
